job/server/src/main/java/alluxio/job/plan/stress/StressBenchDefinition.java

```diff
@@ -123,20 +122,36 @@ public String runTask(StressBenchConfig config, ArrayList<String> args,
     command.add(BaseParameters.DISTRIBUTED_FLAG);
     command.add(BaseParameters.IN_PROCESS_FLAG);
 
-    List<String> commandArgs = config.getArgs().stream().filter((s) ->
-        !BaseParameters.CLUSTER_FLAG.equals(s) && !s.isEmpty())
-        .collect(Collectors.toList());
-
+    List<String> commandArgs = config.getArgs();
+    List<String> newArgs = new ArrayList<>();
     if (commandArgs.stream().anyMatch(
-        (s) -> s.equalsIgnoreCase(UfsIOParameters.USE_MOUNT_CONF))) {
-      // get ufs Uri from --path=blah://blah
-      String ufsUri = commandArgs.stream().filter(s -> !s.startsWith(UfsIOParameters.PATH))
-          .findFirst().map(param -> param.substring("--path".length())).orElse("");
-      commandArgs = commandArgs.stream().filter((s) -> !UfsIOParameters.USE_MOUNT_CONF.equals(s)
-              && !s.startsWith(UfsIOParameters.CONF)).collect(Collectors.toList());
+        (s) -> s.equals(UfsIOParameters.USE_MOUNT_CONF))) {
+      boolean nextElem = false;
+      boolean removeNext = false;
+      String ufsUri = "";
+      for (String elem : commandArgs) {
+        if (elem.equals(UfsIOParameters.CONF)) {
+          removeNext = true;
+        } else {
+          if (!removeNext) {
+            newArgs.add(elem);
+          }
+          removeNext = false;
+        }
+        if (elem.equals(UfsIOParameters.PATH)) {
+          nextElem = true;
+        } else {
+          if (nextElem) {
+            ufsUri = elem;
+            break;
```

Comment:
Is it possible we have more parameters after `PATH`?

Reply:
yes, will fix